client/coral-framework/graphql/mutations/index.js

```diff
@@ -10,16 +10,59 @@ export const postComment = graphql(POST_COMMENT, {
   options: () => ({
     fragments: commentView
   }),
-  props: ({mutate}) => ({
-    postItem: ({asset_id, body, parent_id} /* , type */ ) => {
-      return mutate({
+  props: ({ownProps, mutate}) => ({
+    postItem: ({asset_id, body, parent_id}) =>
+      mutate({
         variables: {
           asset_id,
           body,
           parent_id
+        },
+        optimisticResponse: {
+          createComment: {
+            comment: {
+              user: {
+                id: ownProps.auth.user.id,
+                name: ownProps.auth.user.username
+              },
+              created_at: new Date(),
```

Comment:
I think this needs to be a string? the dates coming back from the json are ISO strings.